mem.go

```diff
@@ -2,12 +2,57 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace uint64
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
 
-	fmt.Println(v)
+func ramOSInfo() (Ram, OS) {
+	v, _ := mem.VirtualMemory()
+	s, _ := mem.SwapMemory()
+	i, _ := host_stat.GetHostInfo()
+	o := OS{i.HostName, i.OSType, i.OSRelease}
+	r := Ram{v.Total, v.Free, s.Free}
+	return r, o
+}
+func printInfo() {
+	mem, os := ramOSInfo()
+	total := mem.total
+	free := mem.free
+	swap := mem.swapSpace
+	totalint := int64(total)
+	totalbig := big.NewInt(totalint)
+	totalmb := humanize.BigBytes(totalbig)
```

Comment:
instead of doing the humanize here, I think it would be better to do that in the functions that create the struct.  The way you have it now, if a user called your function, they would then have to do more work to it.  You printInfo() should just be:

``` go
func printInfo() {
    r, o := ramOSInfo()
    fmt.Println(r)
    fmt.Println(o)
}
```